app/controllers/application_controller.rb

```diff
@@ -1,11 +1,27 @@
 class ApplicationController < ActionController::Base
```

Comment:
The `current_user` method in `ApplicationController` is using `find_by` without any form of authentication. This could potentially allow an attacker to set the `session[:user_id]` to any value and impersonate any user. Ensure that the session management is secure and that user authentication is properly verified.